drivers/ieee802154/Kconfig.nrf5

```diff
@@ -88,4 +88,15 @@ config IEEE802154_NRF5_LOG_RX_FAILURES
 	  It can be helpful for the network traffic analyze but it generates also
 	  a lot of log records in a stress environment.
 
+config IEEE802154_NRF5_MULTIPLE_CCA
+	bool "Support for multiple CCA attempts before transmission"
+	help
+	  This is an optional extension not conforming to IEEE Std. 802.15.4-2015.
```

Comment:
The most recent version of the standard is -2020, maybe it makes sense to reference that for new additions?